networkx/algorithms/triads.py

```diff
@@ -224,6 +249,17 @@ def is_triad(G):
     -------
     istriad : boolean
        Whether G is a valid triad
+
+    Examples
+    --------
+    >>> G = nx.DiGraph()
+    >>> G.add_edges_from([(1, 2), (2, 3), (3, 1)])
+    >>> nx.is_triad(G)
+    True
+    >>> G.add_edges_from([(3, 4), (4, 6)])
```

Comment:
A similar thought here - IMO the example would be every so slightly improved by making it as minimal as possible. In this case, adding a single edge is sufficient to break the triad.
```suggestion
    >>> G.add_edge(0, 1)
```

Reply:
Understood Sir! Committing suggestion.

Reply:
Understood Sir! Committing suggestion.